checker-qual/src/main/java/org/checkerframework/checker/regex/qual/PolyRegex.java

```diff
@@ -11,9 +11,11 @@
 /**
  * A polymorphic qualifier for the Regex type system.
  *
- * <p>Any method written using {@link PolyRegex} conceptually has two versions: one in which every
- * instance of {@link PolyRegex} has been replaced by {@link Regex}, and one in which every instance
- * of {@link PolyRegex} has been replaced by {@link UnknownRegex}.
+ * <p>Any method written using {@link PolyRegex} conceptually has multiple versions: one in which
+ * every instance of {@link PolyRegex} in the method signature has been replaced by one of the
```

Comment:
I reword a little to make it look closer to the original javadoc. Also, I think in **the** method signature is better than in **a** method signature. I am not sure.

Reply:
Resolved